packages/atomic/cypress/e2e/search-box/search-box.cypress.ts

```diff
@@ -473,15 +478,42 @@ describe('Search Box Test Suites', () => {
         .with(addQuerySummary())
         .withoutFirstAutomaticSearch()
         .init();
-      SearchBoxSelectors.inputBox().click();
-      SearchBoxSelectors.inputBox().type('test{enter}', {force: true});
-      SearchBoxSelectors.submitButton().click({force: true});
+      typeSearchInput('test');
     });
 
-    SearchBoxAssertions.assertHasSuggestionsCount(0);
-    SearchBoxAssertions.assertHasText('test');
-    QuerySummaryAssertions.assertHasPlaceholder();
-    CommonAssertions.assertConsoleError(false);
+    it('there are no search suggestions or errors', () => {
+      SearchBoxAssertions.assertHasSuggestionsCount(0);
+      QuerySummaryAssertions.assertHasPlaceholder();
+      CommonAssertions.assertConsoleError(false);
+    });
+  });
+
+  describe('with disableSearchWhenNoQuery set to true', () => {
+    beforeEach(() => {
+      new TestFixture()
+        .with(addSearchBox({props: {'disable-search-when-no-query': 'true'}}))
+        .with(addQuerySummary())
```

Comment:
Is the query summary component needed in this test suite ?

Reply:
looks like its not, will remove - thanks for catching - meant to check after I got the test working.